### ISSUE TITLE
Could not find or load main class c(-Dwebdriver.chrome.driver ...

### ISSUE THREAD

**Issue by pieterjanvc** [ropensci/wdman#33]
Please fill the following template:

### Operating System
Windows 10

### Please describe Expected behaviour
I'm trying to setup a selenium server in R using the provided [vignette](https://cran.r-project.org/web/packages/wdman/vignettes/basics.html)

### Please describe Actual behaviour
I'm getting the following errors
```
x = wdman::selenium(verbose = FALSE, check = FALSE)
x$log()

#$stderr
#[1] "Error: Could not find or load main class c(-Dwebdriver.chrome.driver=\"C:\\\\Users\\\\PJ Van Camp\\\\AppData\\\\Local\\\\binman\\\\binman_chromedriver\\\\win32\\\\111.0.5563.41.chromedriver.exe\","

#$stdout
#character(0)
```

I tried changing the chrome versions, but getting the same error.
I have no idea why this is happening so I'm afraid I cannot provide any more details.
The only thing I thought might be an issue is the space in the path, but it's quoted so that should be OK... 

I also tried running a standalone Selenium server with Java, which is working (if I browse to the IP server address), but then trying to connect to it through R does not work either

```
> remDr <- RSelenium::remoteDriver(
+   remoteServerAddr = "172.31.160.1",
+   port = 4444L,
+   browserName = "chrome"
+ )
> remDr$open()
[1] "Connecting to remote server"
$id
[1] NA
```

Thanks for your help!
PJ

**Comment by mfpole** [comment]
This is the issue described here https://github.com/ropensci/wdman/issues/31, the package is not actively maintained and we cannot update it currently

**Comment by jonathanvoelkle** [comment]
closed in #32
